API/appsettings.Development.json

```diff
@@ -3,12 +3,5 @@
     "DefaultConnection": "Data source=datingapp.db"
   },
   "TokenKey": "This is a super secret key",
```

Comment:
Can add serilog configuration here, and it supports multiple sinkers: have a try a File and Seq